cumulusci/cli/config.py

```diff
@@ -67,18 +67,24 @@ def alert(self, message="We need your attention!"):
             return
         click.echo("\a")
         try:
-            call(
-                [
-                    "osascript",
-                    "-e",
-                    'display notification "{}" with title "{}"'.format(
-                        message.replace('"', r"\"").replace("'", r"\'"), "CumulusCI"
-                    ),
-                ]
-            )
+            call(self._get_platform_alert_cmd(message))
         except OSError:
             pass  # we don't have osascript, probably.
 
+    def _get_platform_alert_cmd(self, message):
+        if sys.platform == "darwin":
+            return [
+                "osascript",
+                "-e",
+                'display notification "{}" with title "{}"'.format(
+                    message.replace('"', r"\"").replace("'", r"\'"), "CumulusCI"
+                ),
+            ]
+        elif sys.platform.startswith("linux"):
+            return ["notify-send", "--icon=utilities-terminal", "CumulusCI", message]
```

Comment:
What are our icon choices? 😅

Reply:
https://standards.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html

I was thinking that `face-cool` ( 😎 ), but wanted to keep it profesh 😏